src/javascripts/background.js

```diff
@@ -0,0 +1,125 @@
+
+/**
+ * Chrome plugin to cycle through tabs.
+ * 
+ * @author Benjamin Oakes <hello@benjaminoakes.com>, @benjaminoakes
```

Comment:
Seems like you wrote a lot of this @madhur :smile:

Let's make sure to give you proper attribution as well

Reply:
Added